Entity/UserRole.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace SumoCoders\FrameworkMultiUserBundle\Entity;
+
+use DateTime;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Security\Core\Role\Role;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table
+ * @ORM\HasLifecycleCallbacks
+ */
+class UserRole extends Role
+{
+    /**
+     * @var int
+     *
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string")
+     */
+    private $roleName;
+
+    /**
+     * @var DateTime
+     *
+     * @ORM\Column(type="datetime")
+     */
+    private $createdOn;
+
+    /**
+     * @var DateTime
+     *
+     * @ORM\Column(type="datetime")
+     */
+    private $editedOn;
+
+    /**
+     * @param string $roleName
+     */
+    public function __construct(string $roleName)
+    {
+        parent::__construct($roleName);
+
+        $this->roleName = $roleName;
+    }
+
+    /**
+     * @return int
```

Comment:
No need to place this PHPDoc here, either. You should remove it wherever possible in this file.